learn/update_and_migration/updating.mdx

```diff
@@ -36,13 +36,124 @@ Once the project has been successfully updated, you will receive an email confir
+
+```sh
+curl \
+  -X POST 'MEILISEARCH_URL/snapshots'
+```
+
+Once the snapshot is available in your [snapshots folder](/learn/self_hosted/configure_meilisearch_at_launch#snapshot-destination), proceed to the next step.
+
+### Step 2: Stop the Meilisearch instance
+
+Next, stop your Meilisearch instance.
+
+<Tabs.Container labels={['Local installation', 'Cloud platforms']}>
+
+<Tabs.Content label="Local installation">
+
+If you're running Meilisearch locally, stop the program by pressing `Ctrl + c`.
+
+</Tabs.Content>
+
+<Tabs.Content label="Cloud platforms">
+
+If you're running Meilisearch as a `systemctl` service, connect via SSH to your cloud instance and execute the following command to stop Meilisearch:
+
+```bash
+systemctl stop meilisearch
+```
+
+You may need to prefix the above command with `sudo` if you are not connected as root.
+
+</Tabs.Content>
+</Tabs.Container>
+
+#### Step 3: Install the new Meilisearch binary
+
+Install the latest version of Meilisearch using:
+
+<Tabs.Container labels={['Local installation', 'Cloud platforms']}>
+
+<Tabs.Content label="Local installation">
+
+```bash
+curl -L https://install.meilisearch.com | sh
+```
+
+</Tabs.Content>
+
+<Tabs.Content label="Cloud platforms">
+
+```sh
+# replace MEILISEARCH_VERSION with the version of your choice. Use the format: `vX.X.X`
+curl "https://github.com/meilisearch/meilisearch/releases/download/MEILISEARCH_VERSION/meilisearch-linux-amd64" --output meilisearch --location --show-error
+```
+
+</Tabs.Content>
+</Tabs.Container>
+
+Give execute permission to the Meilisearch binary:
+
+```
+chmod +x meilisearch
+```
+
+For **cloud platforms**, move the new Meilisearch binary to the `/usr/bin` directory:
+
+```
+mv meilisearch /usr/bin/meilisearch
+```
+
+#### Step 4: Relaunch Meilisearch
+
+Execute the command below to import the dump at launch:
+
+<Tabs.Container labels={['Local installation', 'Cloud platforms']}>
+
+<Tabs.Content label="Local installation">
+
+```bash
+./meilisearch --experimental-dumpless-upgrade 
+```
+
+</Tabs.Content>
+
+<Tabs.Content label="Cloud platforms">
+
+```sh
+meilisearch --experimental-dumpless-upgrade 
+```
+
+</Tabs.Content>
+</Tabs.Container>
+
+Meilisearch should launch normally and immediately create a new `UpgradeDatabase` task. This task is processed immediately and cannot be canceled. You may follow its progress by using the `GET /tasks` endpoint to obtain its `taskUid`, then querying `GET /tasks/TASK_UID`.
+
+While the task is processing, you may continue making search queries. You may also enqueue new tasks. Meilisearch will only process new tasks once `UpgradeDatabase` is completed.
+
+<Capsule intent="warning">
+If after the upgrade is completed and the task status is set to `failed` or Meilisearch returns internal error messages to your queries, [restart your instance from the snapshot](/learn/data_backup/snapshots#starting-from-a-snapshot) you generated during step 1.
```

Comment:
```suggestion
If after the upgrade is completed and the task status is set to `failed` or Meilisearch returns internal error messages to your queries, [restart your instance from the snapshot](/learn/data_backup/snapshots#starting-from-a-snapshot) you generated during step 1 and open an issue to the [Meilisearch repository](https://github.com/meilisearch/meilisearch).
```

Maybe we could also propose to upgrade via a dump? Or it's not needed since it's the category immediately below